CMakeLists.txt

```diff
@@ -223,11 +223,25 @@ if (NOT CMAKE_BUILD_TYPE_UC STREQUAL "RELEASE")
     endif ()
 endif()
 
+if (CMAKE_BUILD_TYPE_UC STREQUAL "RELEASE"
```

Comment:
FWIW I've thought about enabling it for sanitizers build (to get better reports), however the difference is huge, 4.0G vs 6.8G, +70%, so I abandoned this idea.